### PR TITLE
create transactions without skill context

### PR DESCRIPTION
**Pull request by chrispatrick**
# Description
Make it possible to create Transactions without requiring a skill request context.

## Related PRs

None

### CODE REVIEW

File: policy/transact/transact.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+func TransactPolicyResult(
+	ctx context.Context,
+	evalCtx goals.GoalEvaluationContext,
+	configuration skill.Configuration,
+	digest string,
+	previousResult *PreviousResult,
+	evaluationTs time.Time,
+	goalResults []goals.GoalEvaluationQueryResult,
```
Comment (felipecruz91) [inline policy/transact/transact.go:25]: Perhaps we could group the policy-related ones into a struct? This method seems to have an excessive amount of parameters 😅

**Review by felipecruz91 — COMMENTED** [review]
Some minor comments.

**Comment by chrispatrick** [comment]
Yeah, there's some messiness here, but I was mostly copy/pasting. I'll have a look at actually improving things while I'm here.

File: policy/transact/transact.go (hunk 1 above)
Comment (chrispatrick) [reply policy/transact/transact.go:25]: I think it's fine for now. This isn't new. 😄

**Review by felipecruz91 — APPROVED** [review]
LGTM

**Pull request merged**